src/containers/ItemInfo/index.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+
+import './style.css';
+
+class ItemInfo extends Component {
+
+  render() {
+    const { currentItem } = this.props;
```

Comment:
In this case because you only acces the currentItem you could extract everything already from this one.